src/main/webapp/app/exercises/shared/feedback/feedback.component.ts

```diff
@@ -126,13 +126,7 @@ export class FeedbackComponent implements OnInit {
         this.initializeExerciseInformation();
 
         this.feedbackItemService = this.exerciseType === ExerciseType.PROGRAMMING ? this.injector.get(ProgrammingFeedbackItemService) : this.injector.get(FeedbackItemServiceImpl);
-        this.fetchAdditionalInformation();
-
-        if (this.showScoreChart) {
-            this.updateChart(this.feedbackItemNodes);
```

Comment:
This was called before information from `fetchAdditionalInformation` was fetched